internal/lib/stats/memory_metrics.go

```diff
@@ -0,0 +1,169 @@
+package statsserver
+
+import (
+	"github.com/cri-o/cri-o/internal/config/cgmgr"
+	"github.com/cri-o/cri-o/internal/lib/sandbox"
+	"github.com/cri-o/cri-o/internal/oci"
+	"github.com/sirupsen/logrus"
+	types "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func GenerateSandboxMemoryMetrics(sb *sandbox.Sandbox, c *oci.Container, stats interface{}, sm *SandboxMetrics) []*types.Metric {
```

Comment:
Please make private if not used in another package. `stats` can be `*cgmgr.MemoryStats` instead of `interface{}`.